alerts/service.libsonnet

```diff
@@ -0,0 +1,29 @@
+{
+  local prometheusAlertsObj = {
+    groups+: [
+      {
+        name: 'service.rules',
```

Comment:
This should be part of `absent_alerts.libsonnet` of PR ceph/ceph-mixins/pull/50 even if we need this.
Still I feel the other PR is more generic in nature.